system-test/src/systemTest/java/org/apache/mesos/elasticsearch/systemtest/FrameworkRoleSystemTest.java

```diff
@@ -42,8 +42,8 @@ private void testMiniMesosReportsFrameworkRole(String role) throws UnirestExcept
         LOGGER.info("Started Elasticsearch scheduler on " + scheduler.getIpAddress() + ":" + getTestConfig().getSchedulerGuiPort());
 
         ESTasks esTasks = new ESTasks(TEST_CONFIG, scheduler.getIpAddress());
-        new TasksResponse(esTasks, CLUSTER.getConfig().getNumberOfSlaves());
-        new ElasticsearchNodesResponse(esTasks, CLUSTER.getConfig().getNumberOfSlaves());
+        new TasksResponse(esTasks, 3);
+        new ElasticsearchNodesResponse(esTasks, 3);
```

Comment:
3 will be difficult to understand. Should be replaced by cluster.getSlaves().length